server/auth/index.js

```diff
@@ -0,0 +1,45 @@
+import { Op } from 'sequelize';
+import db from '@database/models';
+import { Token } from '@utils/constants';
+
+const getSignedToken = user => new Token({ user }).get();
+
+export const handleSignUp = async (req, res) => {
+  const { firstName, lastName, email, password } = req.body || {};
+  const newUser = await db.users.create({
+    firstName,
+    lastName,
+    email,
+    password
+  });
+  res.data = newUser;
+  const { dataValues } = newUser;
+  res.send({ ...dataValues, token: getSignedToken(newUser) });
+};
+
+export const handleSignIn = async (req, res) => {
+  const { email, password } = req.body || {};
+  const user = await db.users.findOne({
+    where: {
+      [Op.and]: [{ email }, { password }]
```

Comment:
get the user with this email, and then invoke `checkPassword` with the password sent by the user and the hash from the db.